src/Game.cpp

```diff
@@ -261,4 +261,17 @@ bool Game::running() const
 
 bool Game::getWinner() {
 	return this->winner;
+}
+
+void Game::handleCollision(Vector2D& characterPos, SDL_Rect& characterCollider, SDL_Rect& componentCollider)
```

Comment:
- Would change characterPos to entityPos or similar
- Should probably not be in game class but in collider component